meson_options.txt

```diff
@@ -8,7 +8,8 @@ option('enable-uefi', type : 'boolean', value : true, description : 'enable UEFI
 option('enable-uefi-labels', type : 'boolean', value : true, description : 'enable UEFI labels support')
 option('enable-dell', type : 'boolean', value : true, description : 'enable Dell-specific support')
 option('enable-amt', type : 'boolean', value : true, description : 'enable Intel AMT support')
-option('enable-tbtfwu', type : 'boolean', value : true, description : 'enable Thunderbolt support')
+option('enable-tbtfwu', type : 'boolean', value : true, description : 'enable legacy Thunderbolt support')
```

Comment:
As this PR is eventually merged is there really any value in keeping the "legacy" thunderbolt support still in fwupd?  I would think it makes sense to just axe the legacy stuff while pulling in the new.  No distro has actually built the libtbtfwu stuff in, `thunderbolt_icm `never merged into an upstream kernel.  It's just dead code if no one is going to keep building it.

As an illustrative point, it was broke for at least 2 months and no one noticed ;) (https://github.com/hughsie/fwupd/pull/180)